src/index.js

```diff
@@ -1 +1,40 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import getData from "./modules/api.js";
+
+const renderData = (data) => {
+    const itemContainer = document.getElementById("item-container");
+  
+    data.forEach(item => {
+      const card = document.createElement("div");
+      card.classList.add("card");
+  
+      const image = document.createElement("img");
+      image.src = item.imageUrl ? item.imageUrl : "https://via.placeholder.com/150";
+      image.alt = item.title;
+      card.appendChild(image);
```

Comment:
Hi Kazaneza,
This part seems not working. You may use try to use ID for separate shows.